### DISCUSSION TITLE
RDP image cache

### DISCUSSION THREAD

**Discussion by iopasscape** [AndrewRathbun/DFIRArtifactMuseum#17] [Ideas]
Hi there,
Could you add RDP  image cache for Windows please?
Some info here: https://hejelylab.github.io/blog/IRC/RDP-Bitmap-Cache

**Comment by AndrewRathbun** [comment]
I'll try to generate that today or sometime this week. Thanks for the request!

Issue https://github.com/AndrewRathbun/DFIRArtifactMuseum/issues/18

**Comment by iopasscape** [comment]
Awesome, thanks! It would also be great if you generate multiple cache files with different color schemes apart from default 32 bpp one.

**Reply by AndrewRathbun** [reply]
If you have any idea how to do that, let me know. I was just planning on spinning up two VMs, then having them RDP back and forth to each other a few times, and then using KAPE to pull the cache and upload it to the repo. Any thoughts or feedback on that process?

**Comment by iopasscape** [comment]
As far as I remember, you can do it in RDP connection dialog selecting the color scheme fir remote display.

**Reply by AndrewRathbun** [reply]
Ok yeah that sounds familiar. Thanks for the lead on that 👍

**Comment by AndrewRathbun** [comment]
This is done, as seen here: https://github.com/AndrewRathbun/DFIRArtifactMuseum/issues/18
